cfglib.py

```diff
@@ -174,6 +174,8 @@
     cfg.BoolOpt('ignore_empty_images',
                 help='Ignore images with size 0 and exclude them from '
                      'migration process', default=False),
+    cfg.IntOpt('remote_run_attempts', default=5,
+               help='How many attempts after error will be runned'),
```

Comment:
Let's improve the description: `help="Number of retries for remote command execution over SSH"`